{{cookiecutter.project_slug}}/Makefile

```diff
@@ -79,7 +83,7 @@ upload-notebooks:  ### Upload notebooks directory to Storage
 
 .PHONY: download-notebooks
 download-notebooks:  ### Download notebooks directory from Storage
-	neuro cp -r $(NOTEBOOKS_PATH_STORAGE) $(NOTEBOOKS_PATH)
+	neuro cp -r -T $(NOTEBOOKS_PATH_STORAGE) $(NOTEBOOKS_PATH)
```

Comment:
it's a bug

Reply:
True. I've created a related issue (#45).

Reply:
thanks. let's then fix makefile first, and then rebase this branch